actions/lib/helpers.py

```diff
@@ -41,3 +42,10 @@ def get_issue_url(api_client, issue_id):
         pass
 
     return url
+
+
+def convert_timestamps_to_str(dictionary):
```

Comment:
When adding new functionally, adopt Typing. In this case, you return a dict, so `..(dictionary) -> dict:`.